jni/tests/faiss_memory_test.cpp

```diff
@@ -0,0 +1,298 @@
+/*
```

Comment:
Executable name be "memory_test" given it includes nmslib?

Reply:
++ FIXED at https://github.com/opensearch-project/k-NN/pull/1139/commits/85b37c19eb43348a6b1f29c11de7bc678359153b